pkg/scheduler/actions/allocate/allocate.go

```diff
@@ -204,6 +204,10 @@ func (alloc *Action) allocateResources(queues *util.PriorityQueue, jobsMap map[a
 		var stmt *framework.Statement
 		var tasksQueue *util.PriorityQueue
 		if hardMode {
+			if !alloc.session.HyperNodesReadyToSchedule {
+				klog.ErrorS(nil, "HyperNodes not completely populated and not ready to schedule, please check logs for more details", "job", job.UID)
```

Comment:
if error is nil, Is it better to use `Errorf` or `Errorln`? Because ErrorS will print 
```
"HyperNodes not completely populated and not ready to schedule, please check logs for more details" err=<nil> job="example-job-uid"
```

`err=<nil>` is strange

Reply:
`klog.ErrorS` will not print err if err == nil: )

Reply:
yes, you are right. My mistake.